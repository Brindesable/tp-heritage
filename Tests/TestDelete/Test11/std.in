C circle 0 0 10
DELETE circle
C circle 0 0 10
EXIT
