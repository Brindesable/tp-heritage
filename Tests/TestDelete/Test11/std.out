OK
#1 object(s) created
OK
#1 object(s) deleted
OK
#1 object(s) created
